app/src/main/java/org/mozilla/focus/tiles/HomeTilesManager.kt

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.tiles
+
+import android.content.Context
+import android.content.Context.MODE_PRIVATE
+import android.content.SharedPreferences
+import org.json.JSONArray
+import org.json.JSONObject
+
+const val HOME_TILES_PREFS = "hometilesPrefs"
```

Comment:
nit: camel-casing. Also, this is a shared preferences branch so maybe including "Prefs" in the string is not necessary.